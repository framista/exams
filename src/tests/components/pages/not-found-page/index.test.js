import React from 'react';
import NotFoundPage from '../../../../components/pages/not-found-page';
import { shallow } from 'enzyme';

describe('test notFoundPage', () => {
  it('should render NotFoundPage correctly', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
