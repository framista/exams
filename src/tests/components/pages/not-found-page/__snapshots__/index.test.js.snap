// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test notFoundPage should render NotFoundPage correctly 1`] = `
<Connect(PageContainer)>
  <p
    className="mt-5 display-3 text-primary text-center"
  >
    404
  </p>
  <h1
    className="my-5 text-primary text-center text-uppercase"
  >
    Strona niedostępna
  </h1>
  <h3
    className="text-primary text-center"
  >
    Niestety podany przez Ciebie adres strony nie jest dostępny
  </h3>
  <div
    className="mt-5 text-center"
  >
    <Link
      className="text-primary btn btn-primary text-light text-uppercase"
      to="/"
    >
      Strona główna
    </Link>
  </div>
</Connect(PageContainer)>
`;
